Tests/ForageSDKTests/ForageServiceTests.swift

```diff
@@ -205,6 +205,41 @@ final class ForageServiceTests: XCTestCase {
         }
         wait(for: [expectation], timeout: 1.0)
     }
+
+    func test_getJitterAmountInSeconds() {
+        let mockSession = URLSessionMock()
+        let service = LiveForageService(provider: Provider(mockSession))
+        
+        func mockRNG() -> Int {
+            return 1000
+        }
+
+        let jitterAmount = service.jitterAmountInSeconds(using: mockRNG)
+        
+        XCTAssertNotNil(jitterAmount, "Jitter amount should not be nil")
+        XCTAssertTrue(jitterAmount == 1)
+    }
+    
+    func test_waitNextAttempt() {
+        let mockSession = URLSessionMock()
+        let service = LiveForageService(provider: Provider(mockSession))
+        
+        let failureExpectation = XCTestExpectation(description: "Completion called before delay")
+        let successExpectation = XCTestExpectation(description: "Completion called after delay")
+        
+        // Mark the expectation as fulfilled only if it failed
+        failureExpectation.isInverted = true
+        
+        service.waitNextAttempt {
```

Comment:
@djoksimo To remove rng from this function we would need to use mock injection to the `LiveForageService` class and keep a reference to the function passed into the `jitterAmountInSeconds` function. Kinda seems like overkill.